Service/TpayService.php

```diff
@@ -91,6 +98,16 @@ public function getPayment(string $orderId): OrderPaymentInterface
         return $order->getPayment();
     }
 
+    public function getOrderById(string $orderId): OrderInterface
```

Comment:
Czy my w sumie, uzywamy tej metody gdzies poza TpayService ?
Bo widze tylko uzycie ponizej, to mozna by tą metodę dropnąc i uzywać bezpośrednio orderRepository.